.github/workflows/docker-e2e.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow pushes new osmosis-e2e-init-chain docker images on every new tag.
+#
+# On every new `vX.Y.0` tag the following images are pushed:
+#
+# osmolabs/osmosis-e2e-init-chain:X.Y.0    # is pushed
+# osmolabs/osmosis-e2e-init-chain:X.Y      # is updated to X.Y.0
+# osmolabs/osmosis-e2e-init-chain:X        # is updated to X.Y.0
+#
+# All the images above have support for linux/amd64 and linux/arm64.
+#
+# Due to QEMU virtualization used to build multi-platform docker images
+# this workflow might take a while to complete.
+name: Build and Push E2E Init Chain Images
+
+on:
+  push:
+    tags:
+    - 'v[0-9]+.[0-9]+.0' # ignore rc
+
+env:
+  DOCKER_REPOSITORY: osmolabs/osmosis-e2e-init-chain
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Check out the repo
+        uses: actions/checkout@v2
+      - 
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - 
+        name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Docker meta (e2e_init_image)
+        id: meta_e2e_init_image
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.DOCKER_REPOSITORY }}
+          flavor: |
+            latest=false
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+      - 
+        name: Build and push 
+        id: build_push_e2e_init_image
+        uses: docker/build-push-action@v2
+        with:
+          file: tests/e2e/initialization/init.Dockerfile
+          context: .
+          push: true
+          platforms: linux/amd64,linux/arm64
+          build-args: |
+            E2E_SCRIPT_NAME=chain
+          tags: ${{ steps.meta_e2e_init_image.outputs.tags }}
```

Comment:
As stated before, replace this with something like https://github.com/osmosis-labs/osmosis/blob/c19a29d9e5a51baabb08c0efd7cd2b64accda550/.github/workflows/push-docker-images.yml#L89-L92